Specs/DateTime/German/DateTimeModel.json

```diff
@@ -1226,5 +1226,97 @@
     },
     "NotSupported": "javascript, python",
     "Results": []
+  },
+  {
+    "Input": "Lass uns um 18:12 einen Kaffee trinken gehen",
+    "Context": {
+      "ReferenceDateTime": "2019-08-05T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "18:12",
+        "Start": 12,
+        "End": 16,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T18:12",
+              "type": "time",
+              "value": "18:12:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Lass uns um 18 uhr 12 einen Kaffee trinken gehen",
+    "Context": {
+      "ReferenceDateTime": "2019-08-05T00:00:00"
+    },
+    "Results": [
+      {
+        "Text": "18 uhr 12",
+        "Start": 12,
+        "End": 20,
+        "TypeName": "datetimeV2.time",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "T18:12",
+              "type": "time",
+              "value": "18:12:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Lass uns um 18 uhr und 12 einen Kaffee trinken gehen",
```

Comment:
Can you add a test case for "Es ist zehn Uhr." too, to represent the usage of "Uhr" as "o'clock" too?